src/reducers/breakpoints.js

```diff
@@ -50,15 +50,15 @@ function locationMoved(location, newLocation) {
   );
 }
 
-function makeLocationId(location: Location) {
+export function makeLocationId(location: Location) {
   return `${location.sourceId}:${location.line}`;
 }
 
 function allBreakpointsDisabled(state) {
   return state.breakpoints.every(x => x.disabled);
 }
 
-function update(state = State(), action: Action) {
+function update(state: Record<BreakpointsState> = State(), action: Action) {
```

Comment:
I wondered about this and in the end I followed the pattern that seems to be present in majority of the components - component is defined in one place, but exported by default at the end of the file. Do you think it would be better to place export immediately before the reducer?